### ISSUE TITLE
When serializing RichTextUser element, 'invalid additional property: code' in style block

### ISSUE THREAD

**Issue by Wilmarmontano** [soxtoby/SlackNet#196]
As says in the title, the element RichTextUser in the styles block the prop 'code' is not valid.

Here how you can reproduce it.

C# Code:
```
new Message()
        {
            Channel = "<channelId>",
            Blocks = new List<Block>()
            {
                new RichTextBlock()
                {
                    Elements = new List<RichTextElement>()
                    {
                        new RichTextSection()
                        {
                            Elements = new List<RichTextSectionElement>()
                            {
                                new RichTextText()
                                {
                                    Text = "Here some text",
                                    Style = new RichTextStyle()
                                }
                            }
                        },
                        new RichTextList()
                        {
                            Style = RichTextListStyle.ordered.ToString(), 
                            Elements = new List<RichTextElement>(){
                               new RichTextSection()
                              {
                                  Elements = new List<RichTextSectionElement>()
                                  {
                                      new RichTextUser()
                                      {
                                          UserId = "U00000000"
                                      }
                                  }
                              },
                              new RichTextSection()
                              {
                                  Elements = new List<RichTextSectionElement>()
                                  {
                                      new RichTextUser()
                                      {
                                          UserId = "U00000000"
                                      }
                                  }
                              },
                            }
                        }
                    }
                }
            }
        }
```

Serialized Json of the message element:
```
{
    "channel": "C00000000",
    "parse": "none",
    "link_names": false,
    "attachments": [],
    "blocks": [
        {
            "elements": [
                {
                    "elements": [
                        {
                            "text": "Here some text",
                            "style": {
                                "bold": false,
                                "code": false,
                                "italic": false,
                                "strike": false
                            },
                            "type": "text"
                        }
                    ],
                    "type": "rich_text_section"
                },
                {
                    "elements": [
                        {
                            "elements": [
                                {
                                    "user_id": "U00000000",
                                    "style": {
                                        "bold": false,
                                        "code": false,
                                        "italic": false,
                                        "strike": false
                                    },
                                    "type": "user"
                                }
                            ],
                            "type": "rich_text_section"
                        },
                        {
                            "elements": [
                                {
                                    "user_id": "U00000000",
                                    "style": {
                                        "bold": false,
                                        "code": false,
                                        "italic": false,
                                        "strike": false
                                    },
                                    "type": "user"
                                }
                            ],
                            "type": "rich_text_section"
                        }
                    ],
                    "style": "ordered",
                    "indent": 0,
                    "border": 0,
                    "type": "rich_text_list"
                }
            ],
            "type": "rich_text"
        }
    ],
    "unfurl_links": false,
    "unfurl_media": true,
    "reply_broadcast": false
}
```

Slack API response
```
Deserialized JSON: 
      {
        "error": "invalid_blocks",
        "errors": [
          "invalid additional property: code [json-pointer:/blocks/0/elements/1/elements/0/elements/0/style]",
          "invalid additional property: code [json-pointer:/blocks/0/elements/1/elements/1/elements/0/style]"
        ],
        "response_metadata": {
          "messages": [
            "[ERROR] invalid additional property: code [json-pointer:/blocks/0/elements/1/elements/0/elements/0/style]",
            "[ERROR] invalid additional property: code [json-pointer:/blocks/0/elements/1/elements/1/elements/0/style]"
          ]
        }
      }
```

**Comment by soxtoby** [comment]
Thanks for raising this, @Wilmarmontano. Looks like Slack's documentation for rich text has changed a lot since I last saw it, so it seems there's quite a few changes I'll need to make around here. I should have something up in the next few days.

**Comment by soxtoby** [comment]
Should be fixed now in [v0.13.0](https://github.com/soxtoby/SlackNet/releases/tag/v0.13.0).
